[2025-03-16T03:08:52.325+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T03:08:52.338+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection manual__2025-03-16T03:08:48.737962+00:00 [queued]>
[2025-03-16T03:08:52.345+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection manual__2025-03-16T03:08:48.737962+00:00 [queued]>
[2025-03-16T03:08:52.346+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-03-16T03:08:52.356+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): bronze_layer_group.check_minio_connection> on 2025-03-16 03:08:48.737962+00:00
[2025-03-16T03:08:52.363+0000] {standard_task_runner.py:72} INFO - Started process 1065 to run task
[2025-03-16T03:08:52.368+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'bronze_layer_group.check_minio_connection', 'manual__2025-03-16T03:08:48.737962+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp_jm218j9']
[2025-03-16T03:08:52.369+0000] {standard_task_runner.py:105} INFO - Job 69: Subtask bronze_layer_group.check_minio_connection
[2025-03-16T03:08:52.415+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection manual__2025-03-16T03:08:48.737962+00:00 [running]> on host 3174348cc1e9
[2025-03-16T03:08:52.480+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='bronze_layer_group.check_minio_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T03:08:48.737962+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T03:08:48.737962+00:00'
[2025-03-16T03:08:52.481+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T03:08:52.485+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-16T03:08:52.489+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2025-03-16T03:08:52.490+0000] {warnings.py:109} WARNING - <string>:9: UserWarning: AWS Connection (conn_id='minio_conn', conn_type='s3') expected connection type 'aws', got 's3'. This connection might not work correctly. Please use Amazon Web Services Connection type.

[2025-03-16T03:08:52.490+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minio_conn', conn_type='s3') credentials retrieved from login and password.
[2025-03-16T03:08:52.972+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-03-16T03:08:53.054+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T03:08:53.054+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=bronze_layer_group.check_minio_connection, run_id=manual__2025-03-16T03:08:48.737962+00:00, execution_date=20250316T030848, start_date=20250316T030852, end_date=20250316T030853
[2025-03-16T03:08:53.088+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T03:08:53.108+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16T03:08:53.110+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
